MAINTAINERS.yml

```
@@ -0,0 +1,1050 @@
+        - include/crypto/
+        - samples/drivers/crypto/
+    labels:
+        - "area: Crypto / RNG"
+
+"Drivers: DMA":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/dma/
+        - include/dt-bindings/dma/
+    labels:
+        - "area: DMA"
+
+"Drivers: Entropy":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/entropy/
+        - samples/drivers/entropy/
+        - tests/drivers/entropy/
+
+"Drivers: Entropy: OpenISA RV32M1":
+    # Provisional entry, to be corrected
+    collaborators:
+        - MaureenHelm
+    files:
+        - drivers/entropy/*rv32m1*
+
+"Drivers: EEPROM":
+    status: maintained
+    maintainers:
+        - henrikbrixandersen
+    files:
+        - drivers/eeprom/
+        - dts/bindings/mtd/*eeprom*
+        - include/drivers/eeprom.h
+        - tests/drivers/eeprom/
+    labels:
+        - "area: EEPROM"
+
+"Drivers: Ethernet":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - tbursztyka
+    collaborators:
+        - jukkar
+        - pfalcon
+    files:
+        - drivers/ethernet/
+    labels:
+        - "area: Ethernet"
+
+"Drivers: Flash":
+    status: maintained
+    maintainers:
+        - nvlsianpu
+    files:
+        - drivers/flash/
+        - dts/bindings/flash_controller/
+        - include/drivers/flash.h
+        - include/flash.h
+        - samples/drivers/flash_shell/
+    labels:
+        - "area: Flash"
+
+"Drivers: GPIO":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - mnkp
+    files:
+        - doc/reference/peripherals/gpio.rst
+        - drivers/gpio/
+        - include/drivers/gpio/
+        - include/drivers/gpio.h
+        - include/gpio.h
+        - include/dt-bindings/gpio/
+        - tests/drivers/gpio/
+        - samples/drivers/gpio/
+    labels:
+        - "area: GPIO"
+
+"Drivers: HW Info":
+    status: maintained
+    maintainers:
+        - alexanderwachter
+    files:
+        - drivers/hwinfo/
+        - dts/bindings/hwinfo/
+        - include/drivers/hwinfo.h
+        - include/hwinfo.h
+        - tests/drivers/hwinfo/
+    labels:
+        - "area: HWINFO"
+
+"Drivers: IEEE 802.15.4":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - tbursztyka
+    collaborators:
+        - jukkar
+    files:
+        - drivers/ieee802154/
+
+"Drivers: Interrupt controllers":
+    # Provisional entry, to be corrected
+    status: orphaned
+    files:
+        - drivers/interrupt_controller/
+        - dts/bindings/interrupt-controller/
+        - include/drivers/interrupt_controller/
+        - include/dt-bindings/interrupt-controller/
+
+"Drivers: IPM":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - andrewboie
+    files:
+        - drivers/ipm/
+    description: >-
+        Inter-processor mailboxes
+
```

```suggestion
"Drivers: IPM stm32":
    # Provisional entry, to be corrected
    collaborators:
        - arnopo
    files:
        - drivers/ipm/ipm_stm32_ipcc.c
        - dts/bindings/ipm/st,stm32-ipcc-mailbox.yaml
```
@arnopo, can you confirm ?
(Btw, you need to update CODEOWNERS with your new GH handle)

---

Yes Ok for me 
Yes need to update CODEOWNERS : https://github.com/zephyrproject-rtos/zephyr/pull/21228